README.md

```diff
@@ -0,0 +1,41 @@
+# Chingu
+
+OpenGL accelerated 2D game framework for Ruby.
+Builds on the awesome [Gosu](https://github.com/gosu/gosu) (Ruby/C++) which provides all the core functionality.
+It adds simple yet powerful game states, pretty input handling, deployment safe asset-handling, a basic re-usable game object and automations of common task.
+
+[Old readme](README.old.rdoc)
```

Comment:
I'd prefer not to archive the old README like this, it will only get stale, and we have a backup in Git anyway (and in ippa's own repository). Let's just keep the parts that are still up-to-date in this file, like the explanation of core classes, but maybe change the headings to titlecase and add extra # as applicable.

Reply:
This was actually what I was gonna do on the start (and did to some degree), however I thought that it would be a little confusing for people (for some reason), I'll do that as you wish